src/dedup_clinvar.py

```diff
@@ -10,29 +10,31 @@
 # recommended usage:
 # ./dedup_clinvar.py < clinvar_table_sorted.tsv > clinvar_table_dedup.tsv
```

Comment:
This script is no longer really used for deduping. May be better to rename it to something like `group_by_allele.py`